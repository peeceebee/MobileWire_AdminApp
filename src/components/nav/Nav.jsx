import React, { Component } from "react";
import { NavLink } from "react-router-dom";

// SVG imports
import { library } from "@fortawesome/fontawesome-svg-core";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import {
  faCalendarAlt,
  faCogs
} from "@fortawesome/free-solid-svg-icons";

library.add(faCalendarAlt, faCogs); 

export class Nav extends Component {
  render() {
    return (
      <nav>
        <ul className="nav-list">
          <NavLink className="nav-btn" to="/">
          <FontAwesomeIcon className="nav-icon" icon="calendar-alt" />
            Sign-In
          </NavLink>
          <NavLink className="nav-btn" to="/settings">
          <FontAwesomeIcon className="nav-icon" icon="cogs" />
            System Settings 
          </NavLink>
        </ul>
      </nav>
    );
  }
}

export default Nav;
