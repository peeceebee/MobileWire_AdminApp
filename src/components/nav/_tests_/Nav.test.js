import Nav from "../Nav";
import React from "react";
import { shallow } from "enzyme";

describe("Nav", () => {
  let wrapper = shallow(<Nav />);
  it("Should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});

