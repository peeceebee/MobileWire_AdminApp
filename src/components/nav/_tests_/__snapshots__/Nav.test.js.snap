// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Should match the snapshot 1`] = `ShallowWrapper {}`;
